### DISCUSSION TITLE
Editing labels in task page

### DISCUSSION THREAD

**Discussion by keshva23** [cvat-ai/cvat#3949] [Q&A]
We customized the task page such a way that , there is option to add "labels" in the task page under project.

But when we open the task again, we cannot see the added labels while creating the task. But the same created label can be seen , if we make the project_id as null (manually) in the "engine_task" table of the database .
 Any hint please?

Thank you.

**Comment by bsekachev** [comment]
Sorry, I can't understand what you want to say in the second paragraph. Maybe it would be easier if you provide some screenshots and other details.
